infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogEntryWriter.java

```diff
@@ -183,7 +176,7 @@ public long apply(LogReplicationEntryMsg msg) throws ReplicationWriterException
      * that is the last log entry it has played.
      * This is called while the writer enter the log entry sync state.
      *
-     * @param snapshot
+     * @param snapshot the base snapshot that last full sync based on.
```

Comment:
nit -> the base snapshot on which the last full sync was based on.

Reply:
done.